OpenSim/Sandbox/MatlabScripts/pendulum_simulation.m

```diff
@@ -0,0 +1,86 @@
+%  * Copyright (c) 2005-2016 Stanford University and the Authors                *
+%  * Author(s): James Dunne, Thomas Uchida                                      *
+%  * Contributor(s): Chris Dembia                                               *
+%  *                                                                            *
+%  * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+%  * not use this file except in compliance with the License. You may obtain a  *
+%  * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+%  *                                                                            *
+%  * Unless required by applicable law or agreed to in software                 *
+%  * distributed under the License is distributed on an "AS IS" BASIS,          *
+%  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+%  * See the License for the specific language governing permissions and        *
+%  * limitations under the License.                                             *
+%  * -------------------------------------------------------------------------- */
+
+%% Read a model into memory, run a fwd simulation,
+clear all; close all; clc;
+
+import org.opensim.modeling.*
+
+%   public static void test_TableReporter_2() throws java.io.IOException {
+timeInterval = 0.1;
+
+% add a model
+modelFileName = 'double_pendulum_markers.osim';
+model = Model(modelFileName);
+
+%% add a console reporter
+consoleReporter = ConsoleReporter();
+consoleReporter.set_report_time_interval(timeInterval);
+
+% When connecting the outputs, set the alias names 'pin1_anlge' and 'q2'.
+% These will appear as the colomn labels
+consoleReporter.updInput('inputs').connect(model.getCoordinateSet().get(0).getOutput('value'),'pin1_angle');
+consoleReporter.updInput('inputs').connect(model.getCoordinateSet().get(1).getOutput('value'),'q2');
+model.addComponent(consoleReporter);
+
+%% add a table reporter for the pendulum coordiantes
+coordinateReporter= TableReporter();
+coordinateReporter.set_report_time_interval(timeInterval);
+
+% When connecting the outputs, set the alias names 'q1' and 'q2'.
+% These will be the colomn labels when printed to file
+coordinateReporter.updInput('inputs').connect(model.getCoordinateSet().get(0).getOutput('value'),'q1');
+coordinateReporter.updInput('inputs').connect(model.getCoordinateSet().get(1).getOutput('value'),'q2');
+model.addComponent(coordinateReporter);
+
+%% add a Vec3 table reporter for the pendulum markers
+markerReporter= TableReporterVec3();
+markerReporter.set_report_time_interval(timeInterval);
+
+% When connecting the outputs, set the alias names 'marker_1' and 'marker_2'.
+% These will be the colomn labels when printed to file
+markerReporter.updInput('inputs').connect(model.getMarkerSet().get(0).getOutput('location'),'marker_1');
+markerReporter.updInput('inputs').connect(model.getMarkerSet().get(1).getOutput('location'),'marker_2');
```

Comment:
This will work, but now that #1441 is merged, you could also use `markerReporter.addToReport(...)`. No need to change that for this PR; we can make such changes later.